aap_compose_dev.yaml

```diff
@@ -200,12 +227,91 @@ services:
       - ".:/src/galaxy_ng"
     environment:
       <<: *common-env
+    user: root
```

Comment:
All containers must run as `root` for reasons:

- the `Dockerfile.rhel8` sets home to `/app` and if GPG uses that as home `skopeo` can't find the keys, skopeo doesn't allow setting custom gnupghome or custom keyring.
- It is required to be able to install new Python libraries on the run

The only way to avoid running as root would be having a custom Dockerfile for dev, and then there perform the keys, scripts, keyrings setup + creation of permissive virtual env.